compute-hypothetical-feature.mjs

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env node
+import {readFileSync} from 'node:fs';
+
+import {compare} from 'compare-versions';
+
+const caniuse = JSON.parse(readFileSync('caniuse/fulldata-json/data-2.0.json', 'utf8'));
+const historicalBrowserData = JSON.parse(readFileSync('historical-browser-data.json', 'utf8'));
+
+// Sort the data in |historicalBrowserData| so that we can search it by date later.
```

Comment:
I threw this in because I wasn't sure if it was sorted, but I don't think it changed the results. @dfabulich is the data guaranteed to be sorted by timestamp already?